### PR TITLE
chore(sdk): v2 compiler to throw no task defined error

### PR DESCRIPTION
**Pull request by ji-yaqi**

### CODE REVIEW

**Comment by chensun** [comment]
Update release note?

**Comment by ji-yaqi** [comment]
> Update release note?

Done.

File: sdk/RELEASE.md [hunk 1]
```diff
@@ -21,14 +23,6 @@
 
 * Support container environment variable in v2. [\#6515](https://github.com/kubeflow/pipelines/pull/6515)
 
-## Breaking Changes
```
Comment (chensun) [inline sdk/RELEASE.md:24]: Maybe keep these empty sections? That seems how TFX handles the release note.
I don't have a strong opinion though. Up to you.

**Review by chensun — COMMENTED** [review]
/lgtm

Thanks!

File: sdk/RELEASE.md (hunk 1 above)
Comment (ji-yaqi) [reply sdk/RELEASE.md:24]: I see, and I will add a N/A to it

**Comment by chensun** [comment]
/retest

/lgtm
/approve

Thanks!

**Pull request merged**